src/main/java/no/unit/nva/amazon/s3/AbortMultipartUploadHandler.java

```diff
@@ -0,0 +1,141 @@
+package no.unit.nva.amazon.s3;
+
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.AbortMultipartUploadRequest;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static no.unit.nva.amazon.s3.Environment.ALLOWED_ORIGIN_KEY;
+import static no.unit.nva.amazon.s3.Environment.S3_UPLOAD_BUCKET_KEY;
+import static no.unit.nva.amazon.s3.GatewayResponse.ACCESS_CONTROL_ALLOW_ORIGIN;
+import static no.unit.nva.amazon.s3.GatewayResponse.BODY_KEY;
+import static org.apache.http.HttpHeaders.CONTENT_TYPE;
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+import static org.apache.http.HttpStatus.SC_INTERNAL_SERVER_ERROR;
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+import static org.apache.http.HttpStatus.SC_OK;
+import static org.apache.http.entity.ContentType.APPLICATION_JSON;
+
+public class AbortMultipartUploadHandler implements RequestHandler<Map<String, Object>, GatewayResponse> {
+
+    public static final String MULTIPART_UPLOAD_ABORTED_MESSAGE = "Multipart Upload aborted";
+    public static final String PARAMETER_INPUT_KEY = "input";
+    public static final String PARAMETER_BODY_KEY = "body";
+    public static final String PARAMETER_UPLOAD_ID_KEY = "uploadId";
+    public static final String PARAMETER_KEY_KEY = "key";
+
+    public final transient String bucketName;
+    private final transient String allowedOrigin;
+    private static final AmazonS3 s3Client = createAmazonS3Client();
+
+    public AbortMultipartUploadHandler() {
+        this(new Environment());
+    }
+
+    /**
+     * Construct for lambda eventhandler to create an upload request for S3.
+     */
+    public AbortMultipartUploadHandler(Environment environment) {
+        this.allowedOrigin = environment.get(ALLOWED_ORIGIN_KEY).orElseThrow(IllegalStateException::new);
```

Comment:
Can be useful to include which environment variable that is missing in the exception message.

Reply:
Missing environmentvarible is now part of message